src/components/inspectors/FormDatePicker.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="form-group position-relative">
+    <label v-uni-for="name">{{ label }}</label>
```

Comment:
The unique ID mixin requires a matching `v-uni-id` property, which I don't see here, so I think `v-uni-for="name"` can be removed. You could wrap the `label` around `date-picker` to create the same association.